lib/adal.js

```diff
@@ -98,6 +98,9 @@ AuthenticationContext = function (config) {
         ERR_MESSAGES: {
             NO_TOKEN: 'User is not authorized'
         },
+        LOADFRAME_TIMEOUT: '15000',
+        LOADFRAME_PENDING: 'pending',
+        LOADFRAME_TIMEOUT_CODE: 'AADSTSTimeout',
```

Comment:
AAD uses "AADSTS" prefix for errors originated at AAD. And it might not be an AADSTSTimeout, some other error might cause network issues and kill the current requests. Also, we don't use codes for errors in adaljs, we use `adal:` prefix for angular broadcasts though. We can remove this property.